baseplate/lib/log_formatter.py

```diff
@@ -0,0 +1,11 @@
+from pythonjsonlogger import jsonlogger
+
+
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
+    def process_log_record(self, log_record: dict) -> dict:
+        log_record["level"] = log_record.pop("levelname", None)
+        try:
+            log_record["traceID"] = int(log_record.get("threadName", None))
+        except (TypeError, ValueError):
```

Comment:
```suggestion
            log_record["traceID"] = int(log_record["threadName"])
        except (KeyError, ValueError):
```

this feels a bit more clear to me and skips trying to feed a `None` to `int()`